src/collector/LogCollectSimpleControl.js

```diff
@@ -41,8 +41,8 @@ module.exports = class LogCollectSimpleControl extends LogCollectBaseControl {
       }
     }
 
-    if (testState === 'failed' && mochaRunnable?._retries > 0) {
-      testTitle += ` (Attempt ${mochaRunnable?._currentRetry + 1})`
+    if (testState === 'failed' && mochaRunnable && mochaRunnable._retries > 0) {
+      testTitle += ` (Attempt ${mochaRunnable & mochaRunnable._currentRetry + 1})`
```

Comment:
You have a missing & on this line.

Also please pull from master as I have fixed some broken tests.

Reply:
@archfz Thanks! Fixed and updated.